### PR TITLE
Arni

### PR DESCRIPTION
**Pull request by alexarnautu**

### CODE REVIEW

**Comment by dooma** [comment]
@alexarnautu Tu de ce scrii cod?

File: app/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -9,40 +9,17 @@ import {
...
-      {
-        path: 'request-password',
-        component: NbRequestPasswordComponent,
-      },
-      {
-        path: 'reset-password',
-        component: NbResetPasswordComponent,
-      },
-    ],
+  { path: '',
```
Comment (dooma) [inline app/src/app/app-routing.module.ts]: Please insert a `<cr>` here

File: app/src/app/app.module.ts [hunk 2]
```diff
@@ -12,16 +12,22 @@ import { CoreModule } from './@core/core.module';
 
 import { AppComponent } from './app.component';
 import { AppRoutingModule } from './app-routing.module';
+
+import { MainModule } from './user/main.module';
```
Comment (dooma) [inline app/src/app/app.module.ts]: Please don't add spaces around imports. Use Optimise imports feature from your IDE/Editor

File: app/src/app/app.module.ts [hunk 3]
```diff
@@ -12,16 +12,22 @@ import { CoreModule } from './@core/core.module';
...
 
+
 @NgModule({
   declarations: [AppComponent],
   imports: [
     BrowserModule,
     BrowserAnimationsModule,
     HttpModule,
     AppRoutingModule,
+    MainModule,
```
Comment (PhilipCotan) [inline app/src/app/app.module.ts]: Please don't leave more than one space between imports / modules if your purpose is to separate their meaning.

File: app/src/app/user/main.module.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+@NgModule({
+  imports: [
+    ThemeModule,
+  ],
+  declarations: [
+    MainComponent,
+  ],
+})
+export class MainModule {
```
Comment (PhilipCotan) [inline app/src/app/user/main.module.ts]: What is the purpose of MainModule? Isn't it a little bit too abstract? Is it for Dashboard / Home Page / Login? Be more specific.

File: app/src/app/user/main.component.ts [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
```
Comment (PhilipCotan) [inline app/src/app/user/main.component.ts]: The same mention as MainModule (please see the comment below).

**Comment by alexarnautu** [comment]
@dooma Am zis să văd și eu cum e Angularul. :)

File: app/src/app/app.module.ts (hunk 2 above)
Comment (alexarnautu) [reply app/src/app/app.module.ts]: That is the way the app was written before. I think we should be consisdent and maintain the same coding style.

**Pull request merged**